app/Http/Controllers/UserController.php

```diff
@@ -212,4 +213,13 @@ protected function updateDashboard(User $user, $dashboard)
 
         return false;
     }
+
+    public function authlog()
+    {
+        $this->authorize('manage', User::class);
+
+        return view('user.authlog', [
+            'authlog' => AuthLog::orderBy('datetime', 'DESC')->get(),
```

Comment:
Should we limit the results or just assume daily.sh keeps them from being ridiculous?

Reply:
I think it's fine since it's cleant in daily